### DISCUSSION TITLE
Problem using this command

### DISCUSSION THREAD

**Discussion by ghost** [community/community#27418] [Other Feature Feedback, Questions, & Ideas]
remote: Support for password authentication was removed on August 13, 2021.
remote: Please see https://docs.github.com/en/get-started/getting-started-with-git/about-remote-repositories#cloning-with-https-urls for information on currently recommended modes of authentication.
fatal: Authentication failed for 'https://github.com/sailen110797/centralgit.git/'

This is what it is showing when I use the command git push -u origin main and use my username and password.

**Comment by airtower-luna** [comment]
You can't use your password to authorize Git operations (like `git push`). You need to [create a personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) and use that instead of the password when Git asks. Your token needs to have the **repo** scope, plus **workflow** if you use GitHub Actions.

**Reply by abeersani** [reply]
how to do that

**Reply by airtower-luna** [reply]
@abeersani, follow the documentation link above!

**Reply by Kroytor23** [reply]
Благодарю, вы мне очень помогли) ответ был на поверхности , я часов 5 крутил вертел)))

**Comment by Mamulashvili** [comment]
You can generate Personal Access Token like described here: https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token  and authenticate with that

**Comment by ghost** [comment]
You'll have to create a personal access token and used it as the password when pushing

**Reply by Deakhadevi** [reply]
I have removed two-way authentication . Even though I am getting error

**Reply by airtower-luna** [reply]
@Deakhadevi, you have to use a token for Git over HTTPS, no matter if you use 2FA.

**Comment by Tatynaae** [comment]
remote: Support for password authentication was removed on August 13, 2021.
remote: Please see https://docs.github.com/en/get-started/getting-started-with-git/about-remote-repositories#cloning-with-https-urls for information on currently recommended modes of authentication.
fatal: Authentication failed for 'https://github.com/tatynaDev/home-work.git/'
